draft-ietf-quic-spin-exp.md

```diff
@@ -239,14 +239,17 @@ fraction of connections.
 
 The selection process SHOULD be designed such that
 on average the spin bit is disabled for at least one eighth of network paths.
-The random choice SHOULD be dependent
-on the source and destination addresses of the path,
-so that the spin bit is consistently enabled or
-disabled for repeated use of the same path.
+The selection process SHOULD be externally unpredictable but consistent for
+any given combination of source and destination address/port. For instance,
+the implementation might have a static key which it uses to key a pseudorandom
+function over these values and use the output to determine whether to
+send the spin bit. The selection process performed at the beginning
+of the connection SHOULD be applied for all paths used by the connection.
```

Comment:
It's unclear if "selection process" means that different paths can have different spin as long as the PRNG is done the same way, or the connection should reuse the result of the first selection.

Reply:
The question is what to do when a connection migrates to multiple path. There are two options: make a global decision to spin or not for the duration of the connection; or, make an independent decision to spin or not for each path. The text here intend to say, make a global decision.

Reply:
"1/8th of network paths" is unclear - how can an endpoint identify paths? I think what is meant is 1/8th of *peers*